src/__tests__/components/DataRow.test.js

```diff
@@ -0,0 +1,24 @@
+/* globals describe it expect */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import * as React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+
+import { getTheme } from '../../components/services/ThemeContext.js'
+import { DataRowComponent } from '../../components/themed/DataRow.js'
+
+describe('DataRow', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+    const props = {
+      label: 'string',
+      value: 11,
+      // eslint-disable-next-line react/no-unused-prop-types
+      marginRem: [11] | 11,
```

Comment:
This should not have the 'or' sign, it was only part of the type.
You should pass just the number 11.